test/testParachainGovernance.t.sol

```diff
@@ -85,6 +85,22 @@ contract ParachainGovernanceTest is Test {
         assertEq(address(gov.token()), address(token));
```

Comment:
should test registry address, teamMultisig address, owner address

Reply:
owner already tested. Added check for registry and multisig addresses